### ISSUE TITLE
Reason for glslang/MachineIndependent/unistd.h?

### ISSUE THREAD

**Issue by mgadams** [KhronosGroup/glslang#86]
Can anybody shed any light on why there's an empty unistd.h in glslang/MachineIndependent/ ? From a quick grep, I don't see that actually get used anywhere in glslang.

This is causing problems when being built into a test app in an environment where we can't use the CMake build system. This empty file is then getting picked up due to include path sharing, breaking other code which needs a real unistd.h.

Thanks,
//Mark

**Comment by xorgy** [comment]
Seems it's been there since 3DLabs last touched glslang in 2005.

In that import, the README.txt talks about why the header is there.
```
2.  You may have a missing header file, unistd.h.  Create an empty one in
some standard devstudio or SDK system include directory.  (Bison/flex
generate a #include <unistd.h>, this makes them happy.)
```

**Comment by mgadams** [comment]
I think you're correct. The one platform I'd be slightly concerned about would be Windows, and I confirmed that the bison.exe in the tools directory doesn't generate code including unistd.h.

Given that, I've created pull request #88 to remove it.

**Comment by johnkslang** [comment]
I think this is a problem from 2003 that does not exist in 2015.  Fixed with pull request #88.

(Flex is no longer part of glslang.)

I suspect the existing MSVC project files should also be removed.  Only CMake is being maintained.
